### PR TITLE
Fixed typo

### PR DESCRIPTION
**Pull request by phersh**
Corrected spelling of ‘fetaures’ to ‘features’.

### CODE REVIEW

**Comment by avdg** [comment]
Hmm.. this seems to be a read-only repo and they don't accept patches here, see #2.

**Comment by phersh** [comment]
@avdg that conversion references this page on [contributing to V8] (https://code.google.com/p/v8-wiki/wiki/Contributing), which references this [using git page] (https://code.google.com/p/v8-wiki/wiki/UsingGit) which has instructions on sending code for review.

It sounds a bit complicated. Perhaps they should host their code at github or refactor their process to simplify it.

**Comment by avdg** [comment]
Hmm... I'm wondering if they want to invest time in that... I know its a complicated project involving a lot of contributors and lots of code (and sure they don't want bad patches), but at the same time they move a pretty fast peace and it might be happening before we know...

A bot checking patches would be a great step forwards I guess. Way easier to contribute.

**Comment by JohnMHarrisJr** [comment]
Why do people feel the need to submit pull requests to mirrors?

File: src/flag-definitions.h [hunk 1]
```diff
@@ -177,7 +177,7 @@ DEFINE_IMPLICATION(use_strong, strong_mode)
 
 DEFINE_BOOL(es_staging, false, "enable all completed harmony features")
 DEFINE_BOOL(harmony, false, "enable all completed harmony features")
-DEFINE_BOOL(harmony_shipping, true, "enable all shipped harmony fetaures")
+DEFINE_BOOL(harmony_shipping, true, "enable all shipped harmony features")
```
Comment (caitp) [inline src/flag-definitions.h]: Hey, this typo is still present in the codebase, why not submit a CL for this? =)

**Comment by roselatin** [comment]
prepare Mercurial As a first step, you should teach your Mercurial name. So you open the file ~ / .hgrc (or Mercurial.ini in your home directory for Windows) with a text editor and add the ui section (user interaction) with your user name: if You are systems that project members checked or validated changes in the source code, or by using command-line tools to upload files to the project of the "Downloads" tab. Password is: kt2qK3qV5PX4

http://git-scm.com/downloads

http://mercurial.selenic.com/downloads


look more to my blog site here  :  http://mercurial-1.blogspot.com

**Comment by natorion** [comment]
I am closing this pull request because this is just a GitHub mirror. If you want to submit a patch, please upload it to our own code review tool at codereview.chromium.org

**Pull request closed without merging**